src/patternfly/components/ModalBox/examples/modal-box-no-header-example.hbs

```diff
@@ -0,0 +1,21 @@
+{{#> modal-box modal-box--attributes='aria-labelledby="modal-title"'}}
+  {{#> modal-box-body}}
+    {{#> modal-box-close}}
+      {{#> button btnClass="pf-m-action" btnAttributes='aria-label="Close Dialog"'}}
+        <i class="fas fa-times"></i>
+      {{/button}}
+    {{/modal-box-close}}  
+    <p>
+      <span id="modal-title">Lorem ipsum dolor sit</span> amet, consectetur adipisicing elit, sed do eiusmod
```

Comment:
Oh, now I see what you did here. I still think `aria-label` is the way to go. I don't think this is a reliable pattern for consumers to follow, because it's too dependent on the first phrase being usable as the modal title.

BUT, we definitely need an example that illustrates `aria-describedby`. Can we either keep this `<span>` but move the closing tag further down in the paragraph, or move the `id` to the `<p>` instead? 

Actually, both examples are valid solutions—it really depends on how much text is in the body, and how much of that text functions as a modal description. And it would be good to show both of them. What do you think about adding a short paragraph with `<p id="...">` to the "ModalBox Large" example, and keeping the `<span id="...">` in this example? 

For reference, here are more details about [`aria-describedby` in a modal dialog](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal)